### ISSUE TITLE
v1.0.3 does not work (with Ruby 3.3)

### ISSUE THREAD

**Issue by jlw** [midwire/free_zipcode_data#24]
### Expected Behavior

The included executable should run without errors.

### Actual Behavior

```
Starting FreeZipcodeData v1.0.3...
Downloading: US.zip from GeoNames...
Processing '/tmp/zips/US.txt.csv' data, please be patient...                                                                                                         |  ETA: ??:??:??
bundler: failed to load command: free_zipcode_data (/Users/jeremyweathers/SourceCode/api/vendor/bundle/ruby/3.3.0/bin/free_zipcode_data)
/Users/jeremyweathers/SourceCode/api/vendor/bundle/ruby/3.3.0/gems/free_zipcode_data-1.0.3/lib/etl/csv_source.rb:8:in `initialize': wrong number of arguments (given 1, expected 0; required keyword: filename) (ArgumentError)
```

The downloaded tab-delimited source file is converted to a CSV file (US.txt.csv) and an empty DB file (free_zipcode_data.sqlite3) is created.

### Steps to reproduce the problem

---
* Current Version: 1.0.3
* Ruby Version: 3.3.4
* Operating System: macOS Sonoma 14.6.1

- add to Gemfile (I want to periodically pull this data within a Rails 7 app)
- bundle install
- `bundle exec free_zipcode_data --work-dir /tmp --country US`

**Comment by midwire** [comment]
Thanks for this @jlw - I'll do my best to get to it this week.

**Comment by midwire** [comment]
Fixed here: [cbb6b24](https://github.com/midwire/free_zipcode_data/commit/cbb6b244412e2ff8942686c02bdd13c0ce75b9ba) - Released 1.0.4
